source/views/contacts/contact-list.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @flow
+ * All About Olaf
+ * iOS Contact page
+ */
+
+import React from 'react'
+import {SectionList, StyleSheet} from 'react-native'
+import {ListSeparator, ListSectionHeader} from '../components/list'
+import {ListEmpty, ListFooter} from '../components/list'
+import {ContactRow} from './contact-row'
+import {data} from './data'
+import groupBy from 'lodash/groupBy'
+import toPairs from 'lodash/toPairs'
+import * as c from '../components/colors'
+import type {ContactType} from './types'
+
+const AAO_URL = 'https://github.com/StoDevX/AAO-React-Native/issues/new'
+
+const groupContacts = (contacts: ContactType[]) => {
+  const grouped = groupBy(contacts, b => b.category || 'Other')
```

Comment:
We don't need the `||` here, as we're guaranteed through the schema that our objects will have a category.